recipe/deactivate.bat

```diff
@@ -0,0 +1,2 @@
+:: Restore environment variables (if there is anything to restore)
+set "INCLUDE=%INCLUDE_CONDA_NVCC_BACKUP%"
```

Comment:
It'd be better to hide this block behind an if-check for `INCLUDE_CONDA_NVCC_BACKUP` existence.